packages/components/src/entities/SampleHeader.tsx

```diff
@@ -0,0 +1,318 @@
+                                            sampleIds={[sampleId]}
+                                            key="picklist"
+                                            user={user}
+                                        />
+                                    </RequiresPermission>
+                                )}
+                                <RequiresPermission user={user} perms={PermissionTypes.ManageSampleWorkflows}>
+                                    <DisableableMenuItem href={jobCreationHref} operationPermitted={canAddToWorkflow}>
+                                        Create Workflow Job
+                                    </DisableableMenuItem>
+                                </RequiresPermission>
+
+                                {!isMedia && !!StorageMenu && <StorageMenu onUpdate={onUpdate} sampleModel={sampleModel} />}
+
+                                {canPrintLabels && <MenuItem onClick={onPrintLabel}>Print Labels</MenuItem>}
+
+                                {!isMedia && (
+                                    <RequiresPermission user={user} perms={PermissionTypes.Delete}>
+                                        <DisableableMenuItem
+                                            disabledMessage={getSampleDeleteMessage(canDelete, error)}
+                                            onClick={onDeleteSample}
+                                            operationPermitted={canDelete}
+                                        >
+                                            Delete {entityDataType?.nounSingular ?? 'Sample'}
+                                        </DisableableMenuItem>
+                                    </RequiresPermission>
+                                )}
+                                {!isMedia && (
+                                    <RequiresPermission perms={PermissionTypes.CanSeeAuditLog}>
+                                        <MenuItem href={AppURL.create(AUDIT_KEY)
+                                            .addParams({
+                                                eventType: SAMPLE_TIMELINE_AUDIT_QUERY.value,
+                                                'query.sampleid~eq': sampleId
+                                            }).toHref()}>
+                                            View Audit History
+                                        </MenuItem>
+                                    </RequiresPermission>
+                                )}
+                            </ManageDropdownButton>
+                        </span>
+                </RequiresPermission>
+            </PageDetailHeader>
+            {showConfirmDelete && (
+                <EntityDeleteModal
+                    queryModel={sampleModel}
+                    useSelected={false}
+                    beforeDelete={onBeforeDelete}
+                    afterDelete={onAfterDelete}
+                    onCancel={onHideModals}
+                    entityDataType={SampleTypeDataType}
```

Comment:
LKB impl was passing through the entityDataType prop if defined. Should this be doing that?
`entityDataType={entityDataType ?? SampleTypeDataType}`